academic-services/knowledge-exploration-service/how-to-deploy-new-indexes-to-makes.md

```diff
@@ -0,0 +1,169 @@
+
+Now go to your base URL.  Ex: http://contosoMAKES.trafficmanager.net/.  Use your favorite tool to execute queries against the API endpoints and verify the responses and response codes.
+
+> [!NOTE]
+> Traffic Manager by default uses http, not https
+
+## Deploy the new version of MAKES
+
+Follow the deployment instructions for deploying the new version of the MAKES index, see [Create in API instance](get-started-create-api-instances.md).
+
+## Add the new instance of MAKES to your Traffic Manager Profile
+
+1. Open the Azure Portal and search for the Azure Traffic Manager Profiles.
+
+![Search for Traffic Manager profiles in the Azure portal](media/how-to-deploy-new-indexes-to-makes-search-tm-profile.png)
+
+2. Click on the "Traffic Manager Profiles" suggestion from the list.  When the page opens, click on the profile that you have previously created.
+
+3. Click "Endpoints" on the left hand side of the page to open the list of endpoints for this traffic manager profile.
+
+![Click on the "Endpoints" item on the left hand side](media/how-to-deploy-new-indexes-to-makes-select-endpoints.png)
+
+4. Click "Add" at the top of the screen to add a new endpoint.
+
+![Click Add to create an endpoint](media/how-to-deploy-new-indexes-to-makes-click-add-endpoint.png)
+
+3. Apply the following settings:
+
+    | Setting | Value |
+    | --------| ----- |
+    | Type | Azure endpoint |
+    | Name | *Add a descriptive name of your choosing here* |
+    | Target resource type | Public IP Address |
+    | Target resource | Select the name of the public IP address that was created for you MAKES deployment |
+    | Custom Header settings | **Leave Blank**  |
+    | Add as disabled | *checked*  |
+
+9. Click "Ok" to create your new endpoint.
+
+It will take Azure a moment to create your new endpoint, but once completed it will be added to the list of endpoints available in your Traffic Manager Profile.
+
+## Swap MAKES instances in Traffic Manager
+
+When you are ready to bring new traffic to your new MAKES instance you must enable the new instance and disable the old.
+
+1. From the endpoints section of your Traffic Manager Profile, click on the endpoint that you just created above.
+
+2. Move the status toggle from "Disabled" to "Enabled" by clicking on it.
+
+3. Click "Save" at the top.
+
+4. Click on the old endpoint that was serving production traffic.
+
+5. Move the status toggle from "Enabled" to "Disabled" by clicking on it.
+
+6. Click "Save" at the top.
+
+## Verify your service is working as expected
```

Comment:
Move before ## Swap MAKES instances in Traffic Manager to ensure new endpoint is online before swap

Reply:
I'll add checks to the public IP before the swap section, this section verifies the traffic manager URL.

---
In reply to: [376059553](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/102#discussion_r376059553) [](ancestors = 376059553)